docs/UserGuide.md

```diff
@@ -3,20 +3,76 @@ layout: page
 title: User Guide
 ---
 
-TrackO is a **desktop app built for small business owners to help them manage orders and their inventory, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you prefer and are fast at typing, TrackO can get your order and inventory management tasks done faster than traditional GUI apps.
-
-* Table of Contents
-{:toc}
+# Welcome To TrackO's User Guide!
```

Comment:
Hi I was curious about the PR so I decided to take a look

I think it is generally good practice to add an extra line break after declaring a heading with `#`. See: https://google.github.io/styleguide/docguide/style.html#add-spacing-to-headings

I think adding a line break before and after will also help with making sure the GitHub theme renders the headings properly!

Reply:
Thanks for letting me know!